docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,269 @@
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: CLI
+description: Use YugabyteDB Anywhere CLI to create and manage resources in YBA.
+headcontent: Install the CLI, configure default settings, and set up autocompletion
+techPreview: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba.md) (CLI) is an open source tool that enables you to interact with YBA using commands from your shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
+
+The CLI can only be used with YugabyteDB Anywhere v2024.1 or later.
+
+The CLI is in [Tech Preview](/preview/releases/versioning/#feature-availability). Commands and options may change before GA.
+
+## Install CLI
+
+Your YugabyteDB Anywhere installation includes the CLI binary, located in `/opt/yugabyte/yugaware/yba-cli`. The following versions are available:
+
+- yba_cli-<ver>-darwin-amd64
+- yba_cli-<ver>8-darwin-arm64
+- yba_cli-<ver>-linux-amd64
+- yba_cli-<ver>-linux-arm64
+
+Copy the appropriate binary to your local machine. Note that the CLI can only be used with the same or earlier version of YBA.
+
+For help, enter the following command:
+
+```sh
+yba help
+```
+
+### Use CLI
+
+Using the CLI requires providing, at minimum, an [API token](../#authentication).
+
+You can pass the token as a flag when running CLI commands. For example:
+
+```sh
+yba universe list --apiToken "eyJ..."
+```
+
+For convenience, you can configure the CLI with a default API token as follows:
+
+- Use the `auth` command to write the token to a CLI configuration file, as follows:
+
+  ```sh
+  yba auth
+  ```
+
+  At the prompt, paste your API token and press Enter.
+
+  By default, this writes the value to the CLI configuration file `$HOME/.yba-cli.yaml`.
+
+  You can create multiple configuration files, and switch between them using the [--config](#global-flags) flag. You can add any of the other [global flags](#global-flags) to your configuration files.
+
+- Using [environment variables](#environment-variables). Environment variables must begin with `YBA_`. For example:
+
+  ```sh
+  export YBA_APITOKEN=AWERDFSSS
+  yba universe list
+  ```
+
+If a value is set in an environment variable and a configuration file, the environment variable takes precedence. Setting the value using the flag takes precedence over both.
+
+### Environment variables
+
+You can set the following CLI environment variables.
+
+| Variable | Description |
+| :--- | :--- |
+| YBA_CI | Set to `true` to avoid outputting unnecessary log lines. |
+| YBA_AWS_SECRET_KEY | AWS secret access key. For encryption at rest of AWS clusters. |
```

Comment:
This environment variable is not supported via YBA CLI. Could we please remove this?

Reply:
If already not included above, YBA_APITOKEN and YBA_HOST are also env variables that are supported

Reply:
@subramanian-neelakantan commented above that YBA_APITOKEN is not supported...

Reply:
Oh, I see there is another comment where it is discussed. Will add back in